packages/ibm-products/src/components/SteppedAnimatedMedia/_storybook-styles.scss

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright IBM Corp. 2023, 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../global/styles/project-settings';
+
+.stepped-animated-media-stories__viewport {
+  width: carbon--mini-units(50);
```

Comment:
I don't think `carbon--mini-units` is a thing anymore in Carbon v11

Reply:
Correct: just learned this. Use the following instead:
```
@use '@carbon/layout/scss/convert' as *;
...
width: to-rem(pixel value);
```

Reply:
Confirmed: just learned this yesterday. Please try this instead:
```
@use '@carbon/layout/scss/convert' as *;
...
width: to-rem(400px);
```